be/src/common/config.h

```diff
@@ -126,6 +126,9 @@ CONF_Int32(sleep_one_second, "1");
 // The sleep time for five seconds.
 CONF_Int32(sleep_five_seconds, "5");
 
+// The Minimum time to check big queries
+CONF_Int32(min_execute_time, "5");
```

Comment:
Maybe rename it to something like `min_execute_seconds` to indicate the unit off this field?

Reply:
I didn't get the meaning of this parameter from this word...